src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBase.hpp

```diff
@@ -223,6 +231,8 @@ class CompositionalMultiphaseBase : public FlowSolverBase
 
     static constexpr char const * capPressureNamesString() { return "capPressureNames"; }
 
+    static constexpr char const * thermalConductivityNamesString() { return "thermalConductivityNames"; }
+
```

Comment:
Keep in mind that we are trying to remove the need to specify model names in the solvers. Although it's all done in #1682 so I guess this is fine for now.

Reply:
Yes, you are right, I kept the approach used in the current `develop`, I will update this depending on who merges first.